strawberry/arguments.py

```diff
@@ -123,7 +124,7 @@ def __init__(
                             arg.resolve_forward_ref(first)
                         )
 
-    @property
+    @cached_property
```

Comment:
Might be safer to delegate the caching to `StrawberryAnnotation.resolve()` as currently we allow mutability of the underlying `raw_annotation`. Not sure what the use-case would be, but it might confuse some that want to modify annotations at runtime 🧐

Reply:
thanks! I'll update it later or tomorrow 😊

Reply:
@skilkis was just about to comment the same :D